### DISCUSSION TITLE
Changing Soundfont file on Android, doesn't do anything. Am i missing anything?

### DISCUSSION THREAD

**Discussion by shreks7** [CoderLine/alphaTab#1763] [Q&A]
I am trying to load a SoundFont(sf2) file from Android assets folder but I can't make the player use the loaded sound font file - 

```
// Load SoundFont
  val soundFontData = loadAssetAsByteArray(soundFontFileName)
  alphaTabView.api.loadSoundFont(soundFontData, false)
  alphaTabView.api.soundFontLoaded.on {
      Log.i("AlphaTab", "SoundFont loaded successfully")
  }
  
 private fun loadAssetAsByteArray(fileName: String): Uint8Array {
      val assetManager = assets
      assetManager.open(fileName).use { inputStream ->
          return Uint8Array(inputStream.readBytes().asUByteArray())
      }
  }
  
```

I get the log message "SoundFont loaded successfully" but it still plays the default SoundFont file. Any pointers?

**Comment by Danielku15** [comment]
From the code I cannot see something suspicious. Can you share the soundfont so I can look deeper what's happening? You could also try clearing any soundfonts via https://www.alphatab.net/docs/reference/api/resetsoundfonts/
